do.sh

```diff
@@ -34,11 +34,39 @@ ovn_tester=${topdir}/ovn-tester
 EXTRA_OPTIMIZE=${EXTRA_OPTIMIZE:-no}
 USE_OVSDB_ETCD=${USE_OVSDB_ETCD:-no}
 
+DISTRO_IDS=$(awk -F= '/^ID/{print$2}' /etc/os-release)
```

Comment:
Unfortunately distros don't make this easy.

This works on Fedora and Ubuntu but on my RHEL8 `/etc/os-release` I have:
```
ID="rhel"
ID_LIKE="fedora"
```
Which makes the above compute:
```
$ echo $DISTRO_IDS
"rhel" "fedora"
```
I changed it to
```
DISTRO_IDS=$(awk -F= '/^ID=/{print$2}' /etc/os-release | tr -d '"')
```
and this seemed to do the right thing for both Fedora and RHEL.

Reply:
:+1: